test/http-100.js

```diff
@@ -52,7 +52,7 @@ test('error 103 body', (t) => {
       path: '/',
       method: 'GET'
     }, (err) => {
-      t.strictEqual(err.code, 'HPE_INVALID_CONSTANT')
+      t.strictEqual(err.code, 'INVALID_CONSTANT')
```

Comment:
Can't we mutate the error code to match? i.e. prefix `HPE_` so that we are not as breaking?

Reply:
Sure, no problem.